eth/vm/forks/frontier/headers.py

```diff
@@ -85,7 +94,7 @@ def create_frontier_header_from_parent(parent_header, **header_params):
     return header
 
 
-def configure_frontier_header(vm, **header_params):
+def configure_frontier_header(vm: "FrontierVM", **header_params: Any) -> BlockHeader:
```

Comment:
We may be able to tighten the `header_params` with the `mypy_extensions` but that is nothing that needs to be addressed in this PR. Just pointing it out.